mxcube3/ui/components/SampleQueue/HistorySampleItem.js

```diff
@@ -6,17 +6,17 @@ import './app.less';
 export default class HistorySampleItem extends React.Component {
   constructor(props) {
     super(props);
-    this.collapseSample = props.collapseSample.bind(this, props.id);
+    this.collapseSample = props.collapseSample.bind(this, props.data.clientID);
```

Comment:
This change seems to have broken the functionality of collapsing samples. Reverting to the old code fixes it